lms/djangoapps/instructor/tests/test_enrollment.py

```diff
@@ -494,6 +499,81 @@ def test_delete_submission_scores_attempts_children(self):
         self.assertEqual(unrelated_state['brains'], 'zombie')
 
 
+class TestStudentModuleGrading(SharedModuleStoreTestCase):
+    """
+    Tests the effects of student module manipulations
+    on student grades.
+    """
+    @classmethod
+    def setUpClass(cls):
+        super(TestStudentModuleGrading, cls).setUpClass()
+        cls.course = CourseFactory.create()
+        cls.course_key = cls.course.location.course_key
```

Comment:
Nit: why is it necessary to create a member variable for this?  Can't callers just use `self.course.id`?